xml/sle_update_upgrading.xml

```diff
@@ -72,7 +72,9 @@ Check with Simona, if this is supported or needed.
     In the release notes you can find additional information on what has
     changed since the previous release of &sle;. If you are skipping one
     or more Service Packs, also check the release notes of the skipped
-    Service Packs for changes made between the different releases.
+    Service Packs for changes made between each release. The release
+    notes usually only contain the changes from the last to the current
```

Comment:
More precisely: `... usually only contain the changes between two subsequent releases.`?